tests/test_runs/test_run_functions.py

```diff
@@ -950,7 +931,7 @@ def test_initialize_model_from_run(self):
         self.assertEqual(flowS.components["Imputer"].parameters["strategy"], '"most_frequent"')
         self.assertEqual(flowS.components["VarianceThreshold"].parameters["threshold"], "0.05")
 
-    @pytest.mark.flaky()
+    @pytest.mark.skip("Already covered by `test_run_and_upload_randomsearch`")
```

Comment:
I think this is suboptimal. This test explicitly checks the trace, while `test_run_and_upload_randomsearch` checks it only implicitly. Do you think you can add explicit checks to `test_run_and_upload_randomsearch`?

Reply:
Could you actually also remove this function now that we don't need it any more?

Reply:
Sorry I didn't ping - it's removed.